File: AOJ/0505 Questionnaire (アンケート).cs
// 0505 Questionnaire (アンケート)
// http://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=0505
// by Equim on 17-02-08

using System;
using System.Text;
using System.Linq;
using System.Collections.Generic;

class Program
{
    public static void Main(string[] args)
    {
        while (true)
        {
            var Size = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
            if (Size[0] == 0) break;

            var Count = new Dictionary<int, int>();

            for (int i = 0; i < Size[0]; i++)
            {
                var Line = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
                for (int j = 0; j < Size[1]; j++)
                {
                    Count[j + 1] = (Count.ContainsKey(j + 1) ? Count[j + 1] : 0) + Line[j];
                }
            }
            
            Console.WriteLine(string.Join(" ", Count.OrderByDescending(p => p.Value).Select(p => p.Key.ToString()).ToArray()));
        }
    }
}

File: AOJ/0505 Questionnaire (アンケート).js
// 0505 Questionnaire (アンケート)
// http://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=0505
// by Equim on 17-02-08

'use strict';

var raw = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n');

while (true) {
    var size = raw.shift().split(' ').map(Number);
    if (size[0] === 0) {
        break;
    }

    var count = {};
    for (var i = 0; i < size[0]; i++) {
        var line = raw.shift().split(' ').map(Number);
        for (var j = 0; j < size[1]; j++) {
            count[j + 1] = ((j + 1) in count ? count[j + 1] : 0) + line[j];
        }
    }

    // Support Lambda pls QAQ
    console.log(Object.keys(count).sort(function(a, b) { return count[a] === count[b] ? a - b : count[b] - count[a]; }).join(' '));
}
